SemiTruck.cpp

```diff
@@ -0,0 +1,29 @@
+#include "SemiTruck.h"
+
+SemiTruck::SemiTruck(const std::string& n) : Vehicle(n) {}
+
+SemiTruck::~SemiTruck() = default;
+
+SemiTruck::SemiTruck(const SemiTruck&) = default;
+
+SemiTruck& SemiTruck::operator=(const SemiTruck&) = default;
+
+void SemiTruck::toSpeedLimit(int s)
+{
+    if( s > 65 )
+    {
+        std::cout<< name << ": I should slow down!" << std::endl;
+        Vehicle::setSpeed(65);
+    }
+    if( s < 65 )
+    {
+        std::cout<< name << ": I'm going too slow!" << std::endl;
+        Vehicle::setSpeed(65);
+    }
+}
```

Comment:
- Name objects what they are
- Name functions what they do: `toSpeedLimit` is not a good name
- D.R.Y. the `setSpeed` call. 
- use `else-if`
- [ ] revise